homeassistant/components/nederlandse_spoorwegen/sensor.py

```diff
@@ -48,7 +48,7 @@
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the departure sensor."""
 
-    nsapi = ns_api.NSAPI(config.get(CONF_EMAIL), config.get(CONF_PASSWORD))
+    nsapi = ns_api.NSAPI(config.get(CONF_PASSWORD))
```

Comment:
I think you should change to use `CONF_API_KEY` here.

Also, use `config[CONF_API_KEY]` (not `get()`) when the value is required.